We do not need to check the source code for formatting, types, and coverage.

--- pytokens-0.3.0/setup.cfg.orig
+++ pytokens-0.3.0/setup.cfg
@@ -32,11 +32,8 @@
 
 [options.extras_require]
 dev =
-    black
     build
-    mypy
     pytest
-    pytest-cov
     setuptools
     tox
     twine
@@ -45,6 +42,3 @@
 [options.package_data]
 pytokens =
     py.typed
-
-[tool:pytest]
-addopts = --cov --cov-report=term-missing
